manifests/generate.sh

```diff
@@ -135,19 +135,21 @@ ${HELM} template \
 	--set calicoctl.image=$REGISTRY/ctl \
 	--set calicoctl.tag=$CALICO_VERSION
 # The first two lines are a newline and a yaml separator - remove them.
-find ocp/tigera-operator -name "*.yaml" | xargs sed -i -e 1,2d
+find ocp/tigera-operator -name "*.yaml" -print0 | xargs -0 sed -i -e 1,2d
 mv $(find ocp/tigera-operator -name "*.yaml") ocp/ && rm -r ocp/tigera-operator
 
 # Generating the upgrade manifest for OCP.
-# It excludes the CRs (01-*) and the specific BPF files to maintain compatibility with iptables.
-VALUES_FILES=$(ls ocp | grep -v -e '^01-' -e 'cluster-network-operator.yaml' -e '02-configmap-calico-resources.yaml')
+# It excludes files specific to using BPF, CRs (03-cr-*) and CRDs to maintain compatibility with iptables.
```

Comment:
I'm not really sure what "maintain compatibility with iptables" means here - I thought we excluded these files because they are install-time configuration that shouldn't be re-applied?

Reply:
to be 100% honest, I didn't touch this part of the comment, but I definitely should. will update the reasoning there

Reply:
I updated the comment, please take a look